w3-aggregation.md

```diff
@@ -153,8 +129,11 @@ type Offer [OfferDetails]
     "with": "did:web:web3.storage",
     "can": "aggregate/offer",
     "nb": {
-      "offer": { "/": "bafy...many-cars" }, /* dag-cbor CID */
-      "commitmentProof": { "/": "commitment...cars-proof" } /* commitment proof */
+      "offer": { "/": "bafy...many-cars" }, /* dag-cbor CID with offer content */
+      "piece": {
+        "link": { "/": "commitment...aggregate-proof" },
+        "size": 10102020203013342343
+      } /* commitment proof for aggregate */
     }
```

Comment:
I'm hesitant to voice this opinion, yet I wonder if we should omit the `size` which I believe could be determinstically derived from the `pieces`. Unless I'm mistaken existing libraries require to specify `size` (on aggregates specifically) because the remaining space could be filled with `0`s. However it creates space for an error where you may have two different aggregates for exact same pieces.

Reply:
The problem here is that this is not directly compatible with UCAN LOG. When we want to create metrics or to run consumers based on invocations/receipts of these invocations we would need to do extra ops to read block from CAR file. We could make the block as you suggest, which would probably be the exact content we send to Spade in the end, but I would still keep `link` and `size` in the `nb` field so that we can easily track things from the log without a layer of indirection